patches/chrome-installer-mini_installer-chrome.release.patch

```diff
@@ -20,6 +20,7 @@ index 1bbc88ff7dd3545b94d8d5f3dd0ed60cd8c1f093..757657fb76edcf074780436e5617f4a6
 +brave_vpn_wireguard_service.exe: %(VersionDir)s\BraveVpnWireguardService\
 +tunnel.dll: %(VersionDir)s\BraveVpnWireguardService\
 +wireguard.dll: %(VersionDir)s\BraveVpnWireguardService\
++elevation_service.exe: %(VersionDir)s\
```

Comment:
This change causes the `elevation_service.exe` to be packaged and installed. We are currently registering the `Brave Elevation Service` but there's no executable. Now the service will work

Reply:
are we generating symbols for the elevation service? Are they getting packaged up to go to backtrace?

Reply:
> are we generating symbols for the elevation service? Are they getting packaged up to go to backtrace?

PDBs are generated for all binaries. `chrome.release` is used as a list of files which symbol server should be aware of. So yes, with this change we will send one more PDB.

The more interesting question here - how exactly crashpad runs along with elevation_service?

Reply:
@goodov I was pretty sure it initialized crashpad, but now that I look again I don't see anything so maybe it doesn't. I believe the old service did initialize crashpad so maybe we need to add that into our override here